src/privileged/blurts/FirefoxMonitor.jsm

```diff
@@ -0,0 +1,834 @@
+}
+
+let gExtension;
+
+function sha1(str) {
+  console.log(`string at start : ${str}`);
+  let converter =
+    Components.classes["@mozilla.org/intl/scriptableunicodeconverter"].
+      createInstance(Components.interfaces.nsIScriptableUnicodeConverter);
+  converter.charset = "UTF-8";
+  let result = {};
+  let data = converter.convertToByteArray(str, result);
+  console.log(`data result = ${str}`);
+  let ch = Components.classes["@mozilla.org/security/hash;1"]
+                     .createInstance(Components.interfaces.nsICryptoHash);
+  ch.init(ch.SHA1);
+  ch.update(data, data.length);
+  console.log(`ch = ${ch}`);
```

Comment:
Possibly drop all the `console.log` statements

Reply:
hahaha clearly I was having lots of fun trying to figure out why this little hashing snippet was generating different hashes than the one over at blurts-server.